runelite-client/src/main/java/net/runelite/client/plugins/suppliestracker/SuppliesTrackerPlugin.java

```diff
@@ -0,0 +1,604 @@
+)
+@Slf4j
+public class SuppliesTrackerPlugin extends Plugin
+{
+	//Hold Supply Data
+	private static SuppliesTrackerPlugin suppliesTrackerPlugin_instance = null;
+	private static HashMap<Integer, Integer> suppliesUsed = new HashMap<>();
+	private static HashMap<Integer, SuppliesTrackerItemEntry> suppliesEntry = new HashMap<>();
+	private ItemContainer old;
+	private Deque<Integer> itemStack = new ArrayDeque<>();
+	private Deque<Integer> slotStack = new ArrayDeque<>();
+	private Deque<String> typeStack = new ArrayDeque<>();
+	private Deque<net.runelite.api.Item[]> oldInventStack = new ArrayDeque<>();
+	private int ammoId = 0;
+	private int ammoAmount = 0;
+	private int thrownId = 0;
+	private int thrownAmount = 0;
+	private int[] throwingIds = new int[]{BRONZE_DART, IRON_DART, STEEL_DART, BLACK_DART, MITHRIL_DART, ADAMANT_DART, RUNE_DART, DRAGON_DART, BRONZE_KNIFE, IRON_KNIFE, STEEL_KNIFE, BLACK_KNIFE, MITHRIL_KNIFE, ADAMANT_KNIFE, RUNE_KNIFE, BRONZE_THROWNAXE, IRON_THROWNAXE, STEEL_THROWNAXE, MITHRIL_THROWNAXE, ADAMANT_THROWNAXE, RUNE_THROWNAXE, DRAGON_KNIFE, DRAGON_KNIFE_22812, DRAGON_KNIFE_22814, DRAGON_KNIFEP_22808, DRAGON_KNIFEP_22810, DRAGON_KNIFEP , DRAGON_THROWNAXE, CHINCHOMPA_10033, RED_CHINCHOMPA_10034, BLACK_CHINCHOMPA};
+	private boolean ammoLoaded = false;
+	private boolean throwingAmmoLoaded = false;
+	private boolean mainHandThrowing = false;
+	private int mainHand = 0;
+	private SuppliesTrackerPanel panel;
+	private NavigationButton navButton;
+	private String[] RAIDS_CONSUMABLES = new String[]{"xeric's", "elder", "twisted", "revitalisation", "overload", "prayer enhance", "pysk", "suphi", "leckish", "brawk", "mycil", "roqed", "kyren", "guanic", "prael", "giral", "phluxia", "kryket", "murng", "psykk"};
+
+
+	@Inject
+	private ClientToolbar clientToolbar;
+
+	@Inject
+	private ItemManager itemManager;
+
+	@Inject
+	private SpriteManager spriteManager;
+
+	@Inject
+	private Client client;
+
+	@Inject
+	private ScheduledExecutorService executorService;
+
+	@Inject
+	private ClientThread clientThread;
+
+	
+	@Override
+	protected void startUp() throws Exception
+	{
+
+		panel = new SuppliesTrackerPanel(clientThread, itemManager, executorService);
+		suppliesTrackerPlugin_instance = this;
+		final BufferedImage header = ImageUtil.getResourceStreamFromClass(getClass(), "sarabrew.png");
+		panel.loadHeaderIcon(header);
+		final BufferedImage icon = ImageUtil.getResourceStreamFromClass(getClass(), "panel_icon.png");
+
+
+		navButton = NavigationButton.builder()
+			.tooltip("Supplies Tracker")
+			.icon(icon)
+			.priority(5)
+			.panel(panel)
+			.build();
+
+		clientToolbar.addNavigation(navButton);
+	}
+
+	@Override
+	protected void shutDown()
+	{
+		clientToolbar.removeNavigation(navButton);
+	}
+
+	@Subscribe
+	public void onAnimationChanged(AnimationChanged animationChanged) throws ExecutionException
+	{
+
+		if (animationChanged.getActor() == client.getLocalPlayer())
+		{
+			if (animationChanged.getActor().getAnimation() == 1167)
```

Comment:
this constant 1167 should be added as a constant to net.runelite.api.AnimationID and then imported into this plugin